src/ophys_etl/modules/decrosstalk/decrosstalk.py

```diff
@@ -445,7 +419,11 @@ def clean_negative_traces(trace_dict: dc_types.ROISetDict) -> dc_types.ROISetDic
 def run_decrosstalk(signal_plane: DecrosstalkingOphysPlane,
                     ct_plane: DecrosstalkingOphysPlane,
                     cache_dir: str = None, clobber: bool = False,
-                    new_style_output: bool = False) -> Tuple[dict, dc_types.ROISetDict]:  # noqa: E501
+                    new_style_output: bool = False) -> Tuple[dict,
+                                                             Tuple[dc_types.ROISetDict, dc_types.ROISetDict],   # noqa: E501
+                                                             Tuple[dc_types.ROISetDict, dc_types.ROISetDict],   # noqa: E501
+                                                             Tuple[dc_types.ROIEventSet, dc_types.ROIEventSet],  # noqa: E501
+                                                             Tuple[dc_types.ROIEventSet, dc_types.ROIEventSet]]:  # noqa: E501
```

Comment:
you could also appease the linter like:
```
new_style_output: bool = False
) -> Tuple[dict,
                Tuple[dc_types.ROISetDict, dc_types.ROISetDict],
                ...
```

Reply:
This was addressed by the great `noqa` cleanup